### ISSUE TITLE
s2f type global options have method, but no action parameter.

### ISSUE THREAD

**Issue by ILightThings** [pbberlin/struc2frm#1]
I am having an issue where there is no method to direct where the GET or POST request is sent to (example below). The current workaround seems to suppress the form tags, and use another function to build the form tags with the `action` parameter. I feel like I might be missing something.

Generated form:
```HTML
<div class='struc2frm struc2frm-94116000'>
<form name='frmMain'  method='POST' >
[...]
</form>
</div><!-- </div class='struc2frm'... -->
```
Form with action parameter:
```HTML
 <form action="/action_page.php" method="get">
  <label for="fname">First name:</label>
  <input type="text" id="fname" name="fname"><br><br>
  <label for="lname">Last name:</label>
  <input type="text" id="lname" name="lname"><br><br>
  <input type="submit" value="Submit">
</form> 
```

**Comment by pbberlin** [comment]
Will be fixed nexr week

**Comment by pbberlin** [comment]
Fixed with version 1.2.12.
Thanks for your contribution.
